src/pages/workspaces/workspace/Tools.js

```diff
@@ -183,13 +297,18 @@ export const Tools = _.flow(
         })
       ]),
       div({ style: styles.cardContainer(listView) }, [
-        _.map(config => {
-          return h(ToolCard, {
-            onCopy: () => this.setState({ copyingTool: { namespace: config.namespace, name: config.name } }),
-            onDelete: () => this.setState({ deletingTool: { namespace: config.namespace, name: config.name } }),
-            key: `${config.namespace}/${config.name}`, namespace, name, config, listView
-          })
-        }, configs),
+        h(Clickable, {
+          style: { ...styles.card, ...styles.shortCard, color: colors.green[0], fontSize: 18, lineHeight: '22px' },
```

Comment:
The "Find a Tool" box is wider compared to "Create a notebook":
![Screen Shot 2019-03-11 at 11 06 09 AM](https://user-images.githubusercontent.com/2395211/54134110-b6852500-43ed-11e9-9c5a-bb85ce2242f6.png)
![Screen Shot 2019-03-11 at 11 06 21 AM](https://user-images.githubusercontent.com/2395211/54134113-b71dbb80-43ed-11e9-99ce-48c9225b4231.png)

I recommend matching Create a Notebook card's width.